features/org.wso2.carbon.identity.scim2.common.feature/resources/scim2-schema-extension.config

```diff
@@ -570,6 +570,36 @@
 "referenceTypes":[]
 },
 {
+"attributeURI":"urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:forcePasswordReset",
+"attributeName":"forcePasswordReset",
+"dataType":"boolean",
+"multiValued":"false",
+"description":"Temporary claim to invoke forced password reset feature",
+"required":"false",
+"caseExact":"false",
+"mutability":"readWrite",
+"returned":"default",
+"uniqueness":"none",
+"subAttributes":"null",
+"canonicalValues":[],
+"referenceTypes":[]
+},
+{
+"attributeURI":"urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:oneTimePassword",
+"attributeName":"oneTimePassword",
+"dataType":"string",
+"multiValued":"false",
+"description":"One Time Password",
+"required":"false",
+"caseExact":"false",
+"mutability":"readOnly",
+"returned":"default",
```

Comment:
should this be returned on request?

`"returned":"request"`

Reply:
fixed